Directory.Build.props

```diff
@@ -124,35 +124,12 @@
     <DotNetCmd Condition="'$(OS)' == 'Windows_NT'">$(DotNetCmd).exe</DotNetCmd>
   </PropertyGroup>
 
-  <!-- Choose .targets files that come from Arcade rather than from buildtools -->
-  <!-- TODO: Remove when buildtools dependency removed. -->
-  <PropertyGroup>
-    <ExcludeNotSupportedImport>true</ExcludeNotSupportedImport>
-    <ExcludePartialFacadesImport>true</ExcludePartialFacadesImport>
-    <ExcludeApiCompatImport>true</ExcludeApiCompatImport>
-    <ExcludeReferenceAssembliesImport>true</ExcludeReferenceAssembliesImport>
-    <ExcludePackagingImport>true</ExcludePackagingImport>
-    <ExcludePackageLibsImport>true</ExcludePackageLibsImport>
-    <ExcludeCodeAnalysisImport>true</ExcludeCodeAnalysisImport>
-    <ExcludeResourcesImport>true</ExcludeResourcesImport>
-    <ExcludeVersioningImport>true</ExcludeVersioningImport>
-    <ExcludeSigningImport>true</ExcludeSigningImport>
-    <ExcludeDepProjImport>true</ExcludeDepProjImport>
-    <ExcludeResolveContractImport>true</ExcludeResolveContractImport>
-    <ExcludePackageResolveImport>true</ExcludePackageResolveImport>
-    <ExcludeTestsImport>true</ExcludeTestsImport>
-    <ExcludeBinPlaceImport>true</ExcludeBinPlaceImport>
-  </PropertyGroup>
-
   <!-- workaround https://github.com/dotnet/sdk/issues/2288
```

Comment:
Aside, we surely have the fix for this issue now..

Reply:
Nope, we haven't started picking up a new CLI.